src/index.html

```diff
@@ -11,7 +11,144 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post" onsubmit="onSubmit()">
+      <fieldset class="info-sections">
+        <legend>Personal information</legend>
+        <div class="box">
+          <label for="surname-value">
+            Surname:
+            <input type="text" name="surname"
+            id="surname-value" autocomplete="off"
+            required>
+          </label>
+        </div>
+        <div class="box">
+          <label for="name-value">
+            Name:
+            <input type="text" name="name"
+             id="name-value" autocomplete="off"
+             required>
+          </label>
+        </div>
+        <div class="box">
+          <label for="age-value">
+          How old are You?
+            <input type="number" name="age"
+            id="age-value" value="12"
+            min="8" max="100"
+            autocomplete="off" required>
+          </label>
+        </div>
+        <div class="box">
+          <label for="birthday-value">
+            Full date of birth:
+            <input type="date" name="birthday"
+            id="birthday-value" autocomplete="off"
+            max="2021-09-06" required>
+          </label>
+        </div>
+        <div class="box">
+          <label for="terms-agreement-value">
+            I accept the term of the agreement
+            <input type="checkbox" name="terms-agreement"
```

Comment:
Use camelCase for name attribute
```suggestion
            <input type="checkbox" name="termsAgreement"
```